inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -142,6 +142,7 @@ class MKLDNNGraph {
     friend class MKLDNNInferRequest;
     friend class MKLDNNGraphlessInferRequest;
     friend std::shared_ptr<InferenceEngine::ICNNNetwork> dump_graph_as_ie_net(const MKLDNNGraph &graph);
```

Comment:
Do we still need this method? Why don't we use `dump_graph_as_ie_ngraph_net` in the method `dump_graph_as_dot`?

Reply:
I just wanted to keep this method for some time in order to switch to old one in case of some problems. It does not affect binary size since it's removed by linker anyway.

Reply:
Got it